editing/include/tests.js

```diff
@@ -5582,7 +5582,7 @@ function runConformanceTest(browserTest) {
 
             assert_equals(document.execCommand(browserTest[1][i][0], false, browserTest[1][i][1]),
                 expectedExecCommandReturnValues[i]);
-        }, testName + ": execCommand(" + format_value(browserTest[1][i][0]) + ", false, " + format_value(browserTest[1][i][1]) + ") return value");
+        }, testName + ": command #" + (i+1) + " execCommand(" + format_value(browserTest[1][i][0]) + ", false, " + format_value(browserTest[1][i][1]) + ") return value");
```

Comment:
Yes, sub test name stability is important. We compare the result of every subtest with the previous result of that subtest by name. So if the name changes then the expected result is lost. The worst situation (which has happened) is when two tests race to get one of a pair of names (usually foo-1 and foo-2) and the tests have different expected results. That leads to randomness in the test infrastructure.

If you are convinced that the numbers you're adding here are deterministic enough that they won't change when new tests are added, that's fine by me. I haven't studied the existing code enough to have a strong opinion so I trust your judgement given the above constraints.